app/src/main/AndroidManifest.xml

```diff
@@ -23,6 +23,14 @@
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
+        <activity android:name=".UIActivity" android:screenOrientation="landscape"
+            >
+            <intent-filter>
+                <action android:name="android.intent.action.MAIN" />
+
+                <category android:name="android.intent.category.LAUNCHER" />
```

Comment:
Cool, feel free to remove the other one Launcher activity from here now and I'll merge my code into the new one at some point.